live.go

```diff
@@ -0,0 +1,125 @@
+package goinsta
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+type LiveCreateResponse struct {
+	BroadcastID                                   int    `json:"broadcast_id"`
+	UploadUrl                                     string `json:"upload_url"`
+	// MaxTimeInSeconds                              int `json:"max_time_in_seconds"`
```

Comment:
Do you need this commented out code?

Reply:
no, not really. but i've kept it there in case someone needs it.